protostar/commands/test/cheatcodes/cheatcode.py

```diff
@@ -0,0 +1,35 @@
+from abc import abstractmethod
+from typing import Any, Type, cast
+
+from starkware.starknet.core.os.syscall_utils import BusinessLogicSysCallHandler
+
+from protostar.commands.test.starkware.cheatable_carried_state import CheatableCarriedState
+from protostar.commands.test.starkware.cheatable_starknet_general_config import CheatableStarknetGeneralConfig
+
+
+class Cheatcode(BusinessLogicSysCallHandler):
```

Comment:
A cheatcode is not a SysCallHandler. We should pass a modified `BusinessLogicSysCallHandler` that supports middlewares as a dependency to this class.